docs/faq.rst

```diff
@@ -4,23 +4,94 @@ Frequently Asked Questions
 How to ...
 -----------
 
-... Run examples?
+**... Run examples?**
     PARETO project provides examples to run the operational produced water management model
     and the strategic produced water management model (see pareto/case_studies/).
     To run the examples, go to:
 
     * pareto/operatinal_water_management/run_operational_model.py
     * pareto/strategic_water_management/run_strategic_model.py
 
-... Get more help?
+
+**... Get more help?**
     Use the website to `register <https://pareto.org/register/>`_ for the PARETO support mailing list.
     Then you can send questions to |support-email|. For more specific technical questions, we recommend
     our new `PARETO discussions board on Github <https://github.com/project-pareto/discussions>`_.
 
 Troubleshooting
 ---------------
 
-Missing win32api DLL
+**No available solver found among choices: (('cbc',),)**
+
+  If you encounter the error message "No available solver found among choices: (('cbc',),)," 
+  it typically means that the solver 'cbc' is not installed or not correctly configured in 
+  your environment. Here are steps to resolve this issue:
+
+  1. **Solver Installation**
+
+    Ensure that the 'cbc' solver is installed on your system. You can install it using the following methods:
+
+    * **For Windows Users:**
+      The easiest way to get Cbc on Windows is to download an archive from https://github.com/coin-or/Cbc/releases.
```

Comment:
Windows users getting this error message might need a little more help - what should they do after downloading the CBC archive? I think this mostly applies to GUI users - I expect that Python interface PARETO users will obtain CBC by using the `idaes get-extensions` command described in the [Getting Started page](https://pareto.readthedocs.io/en/latest/getting_started/index.html).